docs/pages/packaging.md

```diff
@@ -5,12 +5,29 @@ layout: default
+| [poetry](https://github.com/python-poetry/poetry)                     | A tool that lets you build and package your project as well as managing dependencies                                        | 🟠  |                                                 |
+| [twine](https://pypi.org/project/twine/)                              | Tool for publishing Python packages on PyPI.                                                                                | 🟠  |                                                 |
+| [bump2version](https://pypi.org/project/bump2version/)                | Tool for version-bumping your software. No longer maintained                                                                | 🔴  |                                                 |
+| [bump-my-version](https://github.com/callowayproject/bump-my-version) | Tool for version-bumping your software. Superseded by setuptools_scm                                                        | 🔴  |                                                 |
+
+# Building
+
+| Name                                                  | Short description                                                                                         | 🚦  |                         Used by                          |
+| ----------------------------------------------------- | --------------------------------------------------------------------------------------------------------- | --- | :------------------------------------------------------: |
+| [build](https://pypa-build.readthedocs.io/en/stable/) | Straightforward tool to build a Python package.                                                           | 🟢  |                                                          |
+| [cibuildwheel](https://cibuildwheel.readthedocs.io)   | Builds python wheels for the main operating systems on continuous integration runs (e.g. GitHub actions). | 🟠  | [streamtracer](https://github.com/dstansby/streamtracer) |
+
+## Package configuration file
+
+| Name                                                                                               | Short description                                                                               | 🚦  | Used by |
+| -------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- | --- | :-----: |
+| [pyproject.toml](https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/)             | Contains build system requirements and information, which are used by pip to build the package. It is becoming the accepted standard and we strongly recommend it. | 🟢  |         |
+| [setup.py](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/)  | Strongly coupled with setuptools and therefore not recommended.                                         | 🟠  |         |
```

Comment:
As far as I'm aware sometimes (e.g. with `ext_modules`) a setup.py is unavoidable

Reply:
Yeah, I think it depends on what kind of bindings you have. Pretty sure with `rust` you can avoid it. Also depends on build system of course.

Reply:
So is the suggestion here that `setup.py` should go green? Or that there should be an update to the text and it should stay orange?

Reply:
🟠 / 🟢  depending on whether you need it or not 😅

Reply:
To stop this from going even more stale (it's already months old), I've merged. Please feel encouraged to reopen if there is a wording suggestion here.